app/webpack.config.js

```diff
@@ -0,0 +1,71 @@
+const path = require("path");
```

Comment:
I'd recommend to split your webpack config into 3 different files:
   - webpack.development.config
   - webpack.production.config
   - webpack.common.config
   To include webpack.common config you can use https://github.com/survivejs/webpack-merge